app/src/main/java/com/example/campusguide/BottomNavigation.kt

```diff
@@ -46,4 +42,14 @@ class BottomNavigation constructor (
     private fun startNavigation() {
         directions.startFlow()
     }
+
+    private fun choosePOI() {
+        val permissions = Permissions(activity)
+        activity.permissions = permissions
```

Comment:
Do these have to be 2 different lines?

Reply:
to be honest I copied it from another similar class which is why I did it like that but I think i can make it one line

Reply:
done